components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/internal/util/TemplateManagerHelper.java

```diff
@@ -72,8 +72,7 @@ public static JsonObject fileToJson(File jsonFile) {
             Reader reader = new FileReader(jsonFile);
             jsonObject = gson.fromJson(reader, JsonObject.class);
         } catch (FileNotFoundException e) {
-            //log.error("FileNotFound Exception occurred when converting JSON file to JSON Object", e); //todo: FileNotFound exception occured. error message?
-            log.error(e.getMessage(),e);
+            log.error(e.getMessage(), e);
```

Comment:
give more context and close the reader

Reply:
This is fixed by 3674cdddd78974cf7b8aca6a5feb11440c7c499b